docs/guides/configuration.md

```diff
@@ -40,8 +41,7 @@ Head to the Templates tab and select Add.
 
 Fill out the required fields. The Inventory and Credential dropdowns only show you the items you have access to. You can give this Template another unique name, as well as custom description.
 
-![](../images/home/home-5.png)
-
+![](../images/home/home-5.png) -->
 ### Verifying the template
```

Comment:
change
Verifying the template
to
Configuring IBM i server

Reply:
Done